sqlx-core/src/sqlite/options/vfs.rs

```diff
@@ -0,0 +1,85 @@
+use crate::error::Error;
+use std::str::FromStr;
+
+/// Refer to [SQLite documentation] for available VFSes.
+/// Currently only standard VFSes are supported
+///
+/// [SQLite documentation]: https://www.sqlite.org/vfs.html
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum SqliteVfs {
```

Comment:
I don't think we should be parsing the VFS name because the user may want to pass a custom VFS name. We should just always handle it as a string.